core/src/test_help/mod.rs

```diff
@@ -285,17 +341,31 @@ pub(crate) struct MockPollCfg {
     /// Instead, they will just block forever.
     pub using_rust_sdk: bool,
 }
+#[derive(Copy, Clone)]
+pub(crate) enum ExpectationAmount {
+    Zero,
+    Specific(usize),
+}
```

Comment:
Why not just use `usize`?

Reply:
This went through a few evolutions, but yeah it can just be a usize now